pkgs/development/libraries/qt-5/modules/qtwebengine.nix

```diff
@@ -159,7 +159,7 @@ qtModule {
 
     libevent
   ] ++ optionals (stdenv.hostPlatform.isAarch32 || stdenv.hostPlatform.isAarch64) [
-    ffmpeg_3
+    ffmpeg
```

Comment:
Does anyone know why this is only used on ARM?
Do we use a bundled Ffmpeg on other platforms? If so, we shouldn't.

Reply:
Looking at qmakeFlags above:

```
  qmakeFlags = if stdenv.hostPlatform.isAarch32 || stdenv.hostPlatform.isAarch64
    then [ "--" "-system-ffmpeg" ] ++ optional enableProprietaryCodecs "-proprietary-codecs"
    else optional enableProprietaryCodecs "-- -proprietary-codecs";
```

Maybe we shouldn't? Looks like it was changed in https://github.com/NixOS/nixpkgs/commit/e01c61347e5f5ab030f907a2aaf38d593765b754, but only done for aarch64 to avoid a rebuild.

It's interesting that OfBorg claims this causes 0 rebuilds...

Reply:
> Maybe we shouldn't? Looks like it was changed in [e01c613](https://github.com/NixOS/nixpkgs/commit/e01c61347e5f5ab030f907a2aaf38d593765b754), but only done for aarch64 to avoid a rebuild.

Yes, let;s use system ffmpeg on all platforms.

> It's interesting that OfBorg claims this causes 0 rebuilds...

That's because ofborg only checks on x86_64.

Reply:
Nope, ofborg checks aarch64-linux

 - https://gist.github.com/GrahamcOfBorg/3ed98068e7b993c67ba74677d61834e4

The reported count, though, might be x86_64 only.

Reply:
> The reported count, though, might be x86_64 only.

That's what I meant.